### PR TITLE
Feature: APP-324 APP-325 CTA Cards Carousel

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- Do some fixes on hero and footer for gradient positions and broken CSS
- Add react-responsive-carousel
- Add Carousel and enabled mobile CTA cards

Task: [APP-324](https://aragonassociation.atlassian.net/browse/APP-324)
Task: [APP-325](https://aragonassociation.atlassian.net/browse/APP-325)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/containers/carousel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  } else {
+    return (
+      <MobileCTA>
+        <StyledCarousel
+          swipeable
+          emulateTouch
+          centerMode
+          showArrows={false}
+          showStatus={false}
+          transitionTime={0}
```
Comment (josemarinas) [inline packages/web-app/src/containers/carousel/index.tsx:39]: I would change this from `0` to a higher value like `300` so when you swipe between cards it does not jump

**Review by josemarinas — COMMENTED** [review]
Fine by my side :). The gradient part is a bit tricky to understand, maybe adding some comments would be nice, but don't worry too much about it because this page will be mostly static, so it wouldn't need much change in the future.

File: packages/web-app/src/containers/carousel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  if (isDesktop) {
+    return <DesktopCTA>{ctaList}</DesktopCTA>;
+  } else {
+    return (
+      <MobileCTA>
+        <StyledCarousel
+          swipeable
+          emulateTouch
+          centerMode
+          showArrows={false}
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/carousel/index.tsx:37]: Add  `setShowThumbs = {false}` remove the warning that the carousel is giving.

**Review by Fabricevladimir — APPROVED** [review]
Sweeet! Good work

File: packages/web-app/src/containers/carousel/index.tsx (hunk 1 above)
Comment (sepehr2github) [reply packages/web-app/src/containers/carousel/index.tsx:39]: Did you test it? If you test you'll see the transition will effect the backdrop filter

Comment (josemarinas) [reply packages/web-app/src/containers/carousel/index.tsx:39]: I tested it in Firefox and worked fine, but right now I tested it on Chrome, and it's true that the blur does something weird

**Review by josemarinas — APPROVED** [review]

**Pull request merged**